### PR TITLE
add ddoc main page to bootdoc setup

### PR DESCRIPTION
**Pull request by wilzbach**
an initial overview page - probably needs more love.

@9il - can you fork please fork the `bootDoc` repo, so that I can start making changes to it. Thanks!

### CODE REVIEW

**Comment by 9il** [comment]
> @9il - can you fork please fork the bootDoc repo, so that I can start making changes to it. Thanks!

This is already done by John

**Comment by wilzbach** [comment]
> This is already done by John

Oh didn't see that. Also changed the main page to `index.html` in the settings.

**Comment by wilzbach** [comment]
it looks like this

image

File: index.d [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	$(DT $(DPMODULE combinatorics)
+		$(DD $(DPMODULE2 combinatorics, permutations))
+		$(DD $(DPMODULE2 combinatorics, cartesianPower))
+		$(DD $(DPMODULE2 combinatorics, combinations))
+		$(DD $(DPMODULE2 combinatorics, combinationsRepeat))
+	)
+)
+
+*/
+module mir;
```
Comment (9il) [inline index.d:23]: just index?

**Comment by 9il** [comment]
BTW, we also need a page on readthedocs.org:
I have moved http://d.readthedocs.org/en/latest/ to Andrei

File: index.d (hunk 1 above)
Comment (wilzbach) [inline index.d:23]: the name of the file doesn't matter, but it needs to define `mir` ... :/

**Comment by wilzbach** [comment]
> BTW, we also need a page on readthedocs.org:

ok, but that's for the tutorial or guide then?

**Comment by 9il** [comment]
> ok, but that's for the tutorial or guide then?

Yes, bootDoc for API and readthedocs for ReST guide+ tutorials. Furthermore, we can use `mir.dlang.io/docs` to place ReST using sphinx

**Pull request merged**